plugins/node/opentelemetry-instrumentation-graphql/package.json

```diff
@@ -44,13 +44,15 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.0"
+    "@opentelemetry/api": "^1.0.0",
+    "graphql": "^14.0.0 || ^15.0.0 || ^16.0.0"
   },
   "devDependencies": {
     "@opentelemetry/api": "1.0.2",
     "@opentelemetry/sdk-trace-base": "1.2.0",
     "@types/mocha": "8.2.3",
     "@types/node": "16.11.21",
+    "graphql": "^15.5.1",
```

Comment:
Why not update it among the dev-deps as well to the latest and leave testing on v15 for TAV?

Reply:
Thanks for the great feedback @Rauno56. Those changes are now made.

Reply:
> 

@Rauno56, I finally had some time to get the tests working while using v16 internally. However, since `graphql` v16 doesn't appear to support Node.js v8 or Node.js v10, that has meant conditionally disabling a few of the tests. Perhaps, therefore, you might prefer to use the latest `15.x` internally, so that running `npm test` locally exercises all tests.

BTW the failing code coverage check appears to be as a result of an `as Error` cast I perform, which is weird since casting is only a notional concept, and does not translate to code paths that can be tested.